runtime/doc/starting.txt

```diff
@@ -1168,6 +1168,7 @@ running) you have additional options:
 			file.  This list is read on startup and only changes
```

Comment:
Oi, I rebased after you pushed your commit and merged your line, but forgot to add the `[!]` again. :)